File: app/src/main/java/com/example/leonardomorais/myapplication/MainActivity.java
package com.example.leonardomorais.myapplication;

import android.content.Intent;
import android.support.annotation.NonNull;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

import com.google.android.gms.tasks.OnCompleteListener;
import com.google.android.gms.tasks.Task;
import com.google.firebase.auth.AuthResult;
import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.database.FirebaseDatabase;

public class MainActivity extends AppCompatActivity {

    private EditText email, password;

    private FirebaseAuth auth;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        email = findViewById(R.id.et_email_login);
        password = findViewById(R.id.et_password_login);
    }

    public void onLoginButtonClick(View v){

        auth = FirebaseAuth.getInstance();

        auth.signInWithEmailAndPassword(email.getText().toString(),
                password.getText().toString()).addOnCompleteListener(new OnCompleteListener<AuthResult>() {
            @Override
            public void onComplete(@NonNull Task<AuthResult> task) {
                if(task.isSuccessful()){
                    Intent intent = new Intent(getApplicationContext(), FeedActivity.class);
                    startActivity(intent);
                }
                else{
                    Toast.makeText(MainActivity.this, "errrrrrrrrow", Toast.LENGTH_LONG).show();
                }
            }
        });
    }
}

File: app/src/main/java/com/example/leonardomorais/myapplication/MainActivity.kt
package com.example.leonardomorais.myapplication
import android.content.Intent
import android.support.v4.view.ViewPager
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.content.ContextCompat
import android.view.View

import com.example.leonardomorais.myapplication.Adapter.TabAdapter
import com.example.leonardomorais.myapplication.Helper.SlidingTabLayout
import com.example.leonardomorais.myapplication.R

class MainActivity : AppCompatActivity() {

    private var slidingTabLayout: SlidingTabLayout? = null

    private var viewPager: ViewPager? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val intent = Intent(this, FeedActivity::class.java)
        startActivity(intent)
    }
}
